app/models/manageiq/providers/foreman/configuration_manager.rb

```diff
@@ -31,11 +37,30 @@ def self.description
     @description ||= "Foreman Configuration".freeze
   end
 
+  def provider
+    super || ensure_provider
+  end
+
   def image_name
     "foreman_configuration"
   end
 
   def self.display_name(number = 1)
     n_('Configuration Manager (Foreman)', 'Configuration Managers (Foreman)', number)
   end
+
+  def ensure_provider
```

Comment:
Minor, but should these methods be private?

Reply:
:+1: done